src/components/Header.tsx

```diff
@@ -41,13 +41,16 @@ const MenuLink = ({
   return (
     <RouteLink
       href={href}
-      color="white"
-      borderBottomWidth={isActive ? '4px' : 'none'}
-      borderColor={isActive ? 'primary.500' : 'none'}
+      color="primaryTextLightmode"
+      borderBottomWidth="4px"
+      // TODO: If link is not the active element, borderColor should equal header/drawer background
```

Comment:
Won't be that achievable setting it to none?

`boderColor={isActive ? 'fuchsia.300' : 'none'}`